src/diamond/convertor.py

```diff
@@ -32,47 +51,28 @@ def set(self, value, unit=None):
     def get(self, unit=None):
         return self.do(unit=unit)
 
+    def _object_mapper(self, unit, value, mapping_list, object_type):
+        for counter, units in enumerate(mapping_list):
+            if unit in units:
+                # returning self.type_of_unit
+                return self.convertb(value, object_type, counter)
+        return "None"
```

Comment:
@Lacrymology : Just checked in the failing tests, that the issue isn't due to `convertb` returning `None` but `_object_mapper` returning `None` causes `float(actual_value)` to throw `TypeError`. You can check [here](https://travis-ci.org/python-diamond/Diamond/jobs/125354663) for more details.